run/events-gcs/main_test.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+import main
+
+
+@pytest.fixture
+def client():
+    main.app.testing = True
+    return main.app.test_client()
+
+
+def test_endpoint(client):
```

Comment:
can you add a few tests that we expect to fail?

Reply:
+1.
Returning a string in the HTTP response and testing that the response is accurate would be good. Right now we just test for a 204.

Here is an example of comprehensive tests:
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/hello-broken/main_test.py